### DISCUSSION TITLE
How to change the html language attribute?

### DISCUSSION THREAD

**Discussion by Hkr32** [mkdocs/mkdocs#2352] [Q&A] [answered]
`<html lang="en">`
to
`<html lang="ru">`

**Comment by waylan** [comment, marked as answer]
In short, the answer depends on which theme you are using. Some third-party themes may include an option to change it. However, at this time the built-in themes do not. That said, we are currently working on theme localization/internationalization, which includes the ability to set a language/local. That setting should be reflected in the `lang` attribute of the `html` tag. See #2299 for details.

Outside of waiting for a theme to have official support added, you can change it by creating your own custom theme, whether you fork an existing theme or build your one from scratch.
